src/js/views/globals/app-nav/app-nav_views.js

```diff
@@ -39,7 +39,7 @@ const MainNavDroplist = Droplist.extend({
       lists: [{
         collection: this.collection,
         itemTemplate: hbs`
-          <span class="app-nav__picklist-icon">{{fa iconType icon}}</span>
+          <span class="app-nav__picklist-icon">{{fa iconType icon params}}</span>
```

Comment:
I think `{{fa iconType icon classes="fa-fw"}}` or `{{fa iconType icon classes=params.classes}}` would do this without changing the helper.